.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+# Based on: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
This runs when we add a new git tag in the format `v*`.  This appears to be a somewhat common convention, and maps nicely to how we tag releases.